### PR TITLE
Literal args comma fix

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: src/standard/effectBuilder.html [hunk 1]
```diff
@@ -1,214 +1,269 @@
...
+          effect.static = false;
+        }
+        return arg;
+      }, this);
+      return effect;
+    },
+
+    _parseArg: function(rawArg) {
+      // clean up whitespace
+      var arg = String(rawArg).trim()
```
Comment (kevinpschaaf) [inline src/standard/effectBuilder.html]: Seems like `rawArg` is guaranteed to be a string (comes from `split`)...  seems safe to just do `rawArg.trim(). ...`

File: src/standard/effectBuilder.html [hunk 2]
```diff
@@ -1,214 +1,269 @@
...
+        // replace comma entity with comma
+        .replace(/&comma;/g, ',')
+        // repair escape sequences
+        .replace(/\\(.)/g, '\$1')
```
Comment (kevinpschaaf) [inline src/standard/effectBuilder.html]: We couldn't figure out why this is necessary.

Is this related to your comment in chat?
> you can use \' inside of an attribute, like foo='foo\'bar'
> [2:39 PM] Scott Miles: but the value comes out like: foo\\'bar

Comment (sjmiles) [inline src/standard/effectBuilder.html]: Yes, that is why. If you do 'foo\'bar', the parser will accept it, but it produces a string like 'foo\\'bar', which is an outright bug in the platform IMO. This regex attempts to fix that problem.

**Comment by miztroh** [comment]
From my tests, I'm still having issues with computed expressions that have only literal arguments.  In the example below, node.foobar returns undefined.

```
foobar: {
   computed: "computeFooBar('foo', 'bar')"
}
```

```
computeFooBar(foo, bar) {
   return foo + ' ' + bar;
}
```
